.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,47 @@
+name: Feature request
+description: Suggest an idea for this project
+title: "[ENH] "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **PLEASE READ THIS FIRST:**
+        - DO NOT use the bug and feature tracker for general questions and support requests.
+        Use the `cython-users` mailing list instead.
```

Since links will look nice now, why not add them
```suggestion
        Use the [`cython-users`](https://groups.google.com/g/cython-users) mailing list instead.
```